### DISCUSSION TITLE
Add the ability to save export and import, and save manually.

### DISCUSSION THREAD

**Discussion by ExplodingHairyPotatoCat** [geckoking7/kgt_idle#3] [Ideas]
i can never tell whether i saved recently or not so i wanna be able to save manually

**Comment by geckoking7** [comment]
Well I'm not good enough at coding to do a import/export yet however the game saves whenever you click on inventory or settings, soon it all variables will save automatically. So basic press 2 to save manually.

**Reply by ExplodingHairyPotatoCat** [reply]
Okay, cool. thank you!

**Comment by geckoking7** [comment]
Done :D
